qa/rpc-tests/p2p-versionbits-warning.py

```diff
@@ -90,6 +90,7 @@ def send_blocks_with_version(self, peer, numblocks, nVersionToUse):
             block_time += 1
             height += 1
             tip = block.sha256
+            #tip = block.hash
```

Comment:
oops , that's wasn't a change, it's just one of my comments i had left in there when i was working on a bug.